### PR TITLE
implement Yarn

### PR DESCRIPTION
**Pull request by tcitworld**
This is just a side-project I did when I was bored. It uses Yarn API provided by http://www.weaverdigital.com/
Nothing really important, more fun than useful.

http://thisisyarn.com/

Moreover, you must pay for API access when the requests go over 500.

### CODE REVIEW

File: inc/poche/Poche.class.php [hunk 1]
```diff
@@ -520,6 +520,14 @@ function displayView($view, $id = 0)
                         $flattr = new FlattrItem();
                         $flattr->checkItem($entry['url'], $entry['id']);
                     }
+
+                    # yarn checking
+                    $yarn = NULL;
+                    if (YARN) {
+                        $yarn = new YarnItem();
```
Comment (j0k3r) [inline inc/poche/Poche.class.php]: Form where this class come from?

**Comment by tcitworld** [comment]
Hmm, I guess I forgot to add the file to the commit.

**Comment by ldidry** [comment]
Why am I in this ticket? More important, why is it marked in the history that I closed this pull request on 7 march? I don't have any interest or knowledge in yarn API and I don't have any rights on this repository!

This is strange, isn't it ?

**Comment by tcitworld** [comment]
You had when we tried to use Framasoft's Gitlab and I screwed everything by pushing without checking what I was doing. Nothing to worry about now.

**Comment by ldidry** [comment]
Ok. I understand now. Thx for explanations :-)

**Pull request closed without merging**